tests/test_coverity.py

```diff
@@ -13,9 +14,16 @@
 
 class TestCoverityWarnings(TestCase):
     def setUp(self):
+        os.environ['MIN_COV_WARNINGS'] = '0'

```

Comment:
If you want to test this, then it needs to be different than the values in config file.

Reply:
Done in dcb292dd61197ed79efb7a0b922e9d70f247c3b7